extra/1-currency-conversion.js

```diff
@@ -15,7 +18,13 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(amountGBP) {
+  let amountBRL = (amountGBP * 0.99) * 5.7;
+  amountBRL = Math.round((amountBRL + Number.EPSILON) * 100) / 100;
```

Comment:
Why did you use `Number.EPSILON` here?

Reply:
These have been changed after looking at the solutions folder (for the review).